sdk/communication/communication-calling-server/test/public/serverCall.spec.ts

```diff
@@ -18,6 +22,12 @@ matrix([[true, false]], async function(useAad) {
     let connectionString: string;
 
     beforeEach(async function(this: Context) {
+      // because we have only one test case here, "before each" will hook for the test.
+      // hence, need below block to disable beforeEach block.
+      if (isPlaybackMode()) {
+        // tslint:disable-next-line:no-invalid-this
```

Comment:
why does the parameter need to be named as this? Let's rename the variable instead of putting linter ignores

Reply:
what is the parameter you'd like to rename?

Reply:
maybe context? I think `this` isn't a good variable name anywhere since it's a designated keyword.

Reply:
I have no problem with the renaming proposal. 
But "this" is also used in the template:
https://github.com/Azure/azure-sdk-for-js/blob/0416f8db0b5f0e6d13e497d0316c9f1e6cce2640/sdk/template/template/test/public/configurationClient.spec.ts#L57

Reply:
is there a requirement to keep the name same across the template and the tests?

Reply:
Let's make it clear. this pr is to fix the ci build and live test unblock developers. 

And @richardcho-msft as the new owner for this call automation apis, if we need rename this one, it should be at least same cross this project, which can be done later.